README.md

```diff
@@ -55,6 +55,33 @@ if command -v nix-your-shell > /dev/null; then
 fi
 ```
 
+### Nushell
+
+> **Note**
+> Nushell requires sourced configuration files to exist before `nu` is started.
+
+Add to your `~/.config/nushell/config.nu`:
+
+```nu
+source nix-your-shell.nu
+```
+
+To generate `nix-your-shell.nu` file:
+
+Either manually generate it:
+
+```sh
+nix-your-shell nu | save nix-your-shell.nu
+```
```

Comment:
Maybe we should have a `nix-your-shell nu install` command, which generates the correct configuration and puts it in a place the shell will notice, e.g. appending it to `~/.config/nushell/config.nu`?